en-US/firefox-ios.xliff

```diff
@@ -2903,11 +2848,23 @@
       </trans-unit>
       <trans-unit id="You don’t have any other devices connected to this Firefox Account available to sync.">
         <source>You don’t have any other devices connected to this Firefox Account available to sync.</source>
-        <target>You don't have any other devices connected to this Firefox Account available to sync.</target>
+        <target>You don’t have any other devices connected to this Firefox Account available to sync.</target>
         <note>Error message shown in the remote tabs panel</note>
       </trans-unit>
     </body>
   </file>
+  <file original="Client/UseCustomAccountService.strings" source-language="en" datatype="plaintext" target-language="en">
+    <header>
+      <tool tool-id="com.apple.dt.xcode" tool-name="Xcode" tool-version="8.3.3" build-num="8E3004b"/>
+    </header>
+    <body>
+      <trans-unit id="Use Custom Account Service">
```

Comment:
Needs a proper ID.

Reply:
We have "unexported" all strings related to this feature. This is currently only exposed as a secret/hidden debug feature. The plan is to introduce an official "Advanced" settings section where this can live, and then also make these strings go though official copy.

https://github.com/mozilla-mobile/firefox-ios/pull/3291